src/components/Vocal.js

```diff
@@ -171,8 +210,8 @@ Vocal.defaultProps = {
 	lang: 'en-US',
 	grammars: null,
 	timeout: 3000,
-	ariaLabel: 'speech',
-	tabIndex: -1,
+	ariaLabel: 'microphone icon',
```

Comment:
Par défaut c'est un bouton, donc le ariaLabel doit indiquer l'action effectuée par le bouton en l'absence d'un texte (-> "start recognition"). Ce texte doit pouvoir être surchargé pour permettre un usage dans un contexte i18n.

Si tu as des children, en revanche, c'est eux qui portent les rôles ARIA et qui sont la responsabilité du développeur. Donc dans ce cas, Vocal est un simple container vDOM et ne devrait pas avoir de Aria Label.

Reply:
Ok for the default value. Other issues work already as described.